diff --git a/app/src/main/ic_launcher-playstore.png b/app/src/main/ic_launcher-playstore.png
index 7a81d18c..a6d00bdb 100644
Binary files a/app/src/main/ic_launcher-playstore.png and b/app/src/main/ic_launcher-playstore.png differ
diff --git a/app/src/main/res/mipmap-hdpi/ic_launcher.png b/app/src/main/res/mipmap-hdpi/ic_launcher.png
index 9d3f833c..53abe3ce 100644
Binary files a/app/src/main/res/mipmap-hdpi/ic_launcher.png and b/app/src/main/res/mipmap-hdpi/ic_launcher.png differ
diff --git a/app/src/main/res/mipmap-hdpi/ic_launcher_background.png b/app/src/main/res/mipmap-hdpi/ic_launcher_background.png
index 305c2833..3101bd85 100644
Binary files a/app/src/main/res/mipmap-hdpi/ic_launcher_background.png and b/app/src/main/res/mipmap-hdpi/ic_launcher_background.png differ
diff --git a/app/src/main/res/mipmap-hdpi/ic_launcher_foreground.png b/app/src/main/res/mipmap-hdpi/ic_launcher_foreground.png
index 89317ccf..8953f473 100644
Binary files a/app/src/main/res/mipmap-hdpi/ic_launcher_foreground.png and b/app/src/main/res/mipmap-hdpi/ic_launcher_foreground.png differ
diff --git a/app/src/main/res/mipmap-hdpi/ic_launcher_round.png b/app/src/main/res/mipmap-hdpi/ic_launcher_round.png
index ab04057c..42998c9f 100644
Binary files a/app/src/main/res/mipmap-hdpi/ic_launcher_round.png and b/app/src/main/res/mipmap-hdpi/ic_launcher_round.png differ
diff --git a/app/src/main/res/mipmap-mdpi/ic_launcher.png b/app/src/main/res/mipmap-mdpi/ic_launcher.png
index 932223b3..10712faa 100644
Binary files a/app/src/main/res/mipmap-mdpi/ic_launcher.png and b/app/src/main/res/mipmap-mdpi/ic_launcher.png differ
diff --git a/app/src/main/res/mipmap-mdpi/ic_launcher_background.png b/app/src/main/res/mipmap-mdpi/ic_launcher_background.png
index 0bd0849b..c2125928 100644
Binary files a/app/src/main/res/mipmap-mdpi/ic_launcher_background.png and b/app/src/main/res/mipmap-mdpi/ic_launcher_background.png differ
diff --git a/app/src/main/res/mipmap-mdpi/ic_launcher_foreground.png b/app/src/main/res/mipmap-mdpi/ic_launcher_foreground.png
index 17d3c583..53d0e8b4 100644
Binary files a/app/src/main/res/mipmap-mdpi/ic_launcher_foreground.png and b/app/src/main/res/mipmap-mdpi/ic_launcher_foreground.png differ
diff --git a/app/src/main/res/mipmap-mdpi/ic_launcher_round.png b/app/src/main/res/mipmap-mdpi/ic_launcher_round.png
index a4195022..d211b57a 100644
Binary files a/app/src/main/res/mipmap-mdpi/ic_launcher_round.png and b/app/src/main/res/mipmap-mdpi/ic_launcher_round.png differ
diff --git a/app/src/main/res/mipmap-xhdpi/ic_launcher.png b/app/src/main/res/mipmap-xhdpi/ic_launcher.png
index 7c58c603..73163a5c 100644
Binary files a/app/src/main/res/mipmap-xhdpi/ic_launcher.png and b/app/src/main/res/mipmap-xhdpi/ic_launcher.png differ
diff --git a/app/src/main/res/mipmap-xhdpi/ic_launcher_background.png b/app/src/main/res/mipmap-xhdpi/ic_launcher_background.png
index 4aff2408..3fa582eb 100644
Binary files a/app/src/main/res/mipmap-xhdpi/ic_launcher_background.png and b/app/src/main/res/mipmap-xhdpi/ic_launcher_background.png differ
diff --git a/app/src/main/res/mipmap-xhdpi/ic_launcher_foreground.png b/app/src/main/res/mipmap-xhdpi/ic_launcher_foreground.png
index 3808e9fe..2af02353 100644
Binary files a/app/src/main/res/mipmap-xhdpi/ic_launcher_foreground.png and b/app/src/main/res/mipmap-xhdpi/ic_launcher_foreground.png differ
diff --git a/app/src/main/res/mipmap-xhdpi/ic_launcher_round.png b/app/src/main/res/mipmap-xhdpi/ic_launcher_round.png
index d9053942..56f455f1 100644
Binary files a/app/src/main/res/mipmap-xhdpi/ic_launcher_round.png and b/app/src/main/res/mipmap-xhdpi/ic_launcher_round.png differ
diff --git a/app/src/main/res/mipmap-xxhdpi/ic_launcher.png b/app/src/main/res/mipmap-xxhdpi/ic_launcher.png
index 2edf5ea1..b6ea6365 100644
Binary files a/app/src/main/res/mipmap-xxhdpi/ic_launcher.png and b/app/src/main/res/mipmap-xxhdpi/ic_launcher.png differ
diff --git a/app/src/main/res/mipmap-xxhdpi/ic_launcher_background.png b/app/src/main/res/mipmap-xxhdpi/ic_launcher_background.png
index 9580348a..8f27b684 100644
Binary files a/app/src/main/res/mipmap-xxhdpi/ic_launcher_background.png and b/app/src/main/res/mipmap-xxhdpi/ic_launcher_background.png differ
diff --git a/app/src/main/res/mipmap-xxhdpi/ic_launcher_foreground.png b/app/src/main/res/mipmap-xxhdpi/ic_launcher_foreground.png
index 3cfdbe87..2e8f01eb 100644
Binary files a/app/src/main/res/mipmap-xxhdpi/ic_launcher_foreground.png and b/app/src/main/res/mipmap-xxhdpi/ic_launcher_foreground.png differ
diff --git a/app/src/main/res/mipmap-xxhdpi/ic_launcher_round.png b/app/src/main/res/mipmap-xxhdpi/ic_launcher_round.png
index 36b5cc38..307313cf 100644
Binary files a/app/src/main/res/mipmap-xxhdpi/ic_launcher_round.png and b/app/src/main/res/mipmap-xxhdpi/ic_launcher_round.png differ
diff --git a/app/src/main/res/mipmap-xxxhdpi/ic_launcher.png b/app/src/main/res/mipmap-xxxhdpi/ic_launcher.png
index 37995027..0cfcdea7 100644
Binary files a/app/src/main/res/mipmap-xxxhdpi/ic_launcher.png and b/app/src/main/res/mipmap-xxxhdpi/ic_launcher.png differ
diff --git a/app/src/main/res/mipmap-xxxhdpi/ic_launcher_background.png b/app/src/main/res/mipmap-xxxhdpi/ic_launcher_background.png
index 681d4962..308d9480 100644
Binary files a/app/src/main/res/mipmap-xxxhdpi/ic_launcher_background.png and b/app/src/main/res/mipmap-xxxhdpi/ic_launcher_background.png differ
diff --git a/app/src/main/res/mipmap-xxxhdpi/ic_launcher_foreground.png b/app/src/main/res/mipmap-xxxhdpi/ic_launcher_foreground.png
index 2b32c9b9..2c7b5da6 100644
Binary files a/app/src/main/res/mipmap-xxxhdpi/ic_launcher_foreground.png and b/app/src/main/res/mipmap-xxxhdpi/ic_launcher_foreground.png differ
diff --git a/app/src/main/res/mipmap-xxxhdpi/ic_launcher_round.png b/app/src/main/res/mipmap-xxxhdpi/ic_launcher_round.png
index c0ee1de2..1600e233 100644
Binary files a/app/src/main/res/mipmap-xxxhdpi/ic_launcher_round.png and b/app/src/main/res/mipmap-xxxhdpi/ic_launcher_round.png differ
diff --git a/core/common/src/main/java/com/looker/core/common/extension/Context.kt b/core/common/src/main/java/com/looker/core/common/extension/Context.kt
index 69a0dcfe..9fae7fcc 100644
--- a/core/common/src/main/java/com/looker/core/common/extension/Context.kt
+++ b/core/common/src/main/java/com/looker/core/common/extension/Context.kt
@@ -9,6 +9,7 @@ import androidx.annotation.AttrRes
 import androidx.annotation.DrawableRes
 import androidx.core.content.ContextCompat
 import androidx.core.content.res.ResourcesCompat
+import com.leos.core.common.R
 import com.leos.core.common.nullIfEmpty
 
 inline val Context.notificationManager: NotificationManager
@@ -34,25 +35,30 @@ fun Context.getDrawableFromAttr(attrResId: Int): Drawable {
 	return getDrawableCompat(resId)
 }
 
-fun Context.getDrawableCompat(@DrawableRes resId: Int): Drawable =
+fun Context.getDrawableCompat(@DrawableRes resId: Int = R.drawable.background_border): Drawable =
 	ResourcesCompat.getDrawable(resources, resId, theme) ?: ContextCompat.getDrawable(this, resId)!!
 
 fun Intent.getPackageName(): String? {
-	val uri = data ?: return null
-	val scheme = uri.scheme ?: return null
-	val host = uri.host ?: return null
+	val uri = data
 	return when {
-		scheme == "package" || scheme == "fdroid.app" -> {
+		uri?.scheme == "package" || uri?.scheme == "fdroid.app" -> {
 			uri.schemeSpecificPart?.nullIfEmpty()
 		}
-		scheme == "market" && uri.host == "details" -> {
+		uri?.scheme == "market" && uri.host == "details" -> {
 			uri.getQueryParameter("id")?.nullIfEmpty()
 		}
-		uri.scheme in setOf("http", "https") -> {
-			if (host == "f-droid.org" || host.endsWith(".f-droid.org") || host == "apt.izzysoft.de") {
+		uri != null && uri.scheme in setOf("http", "https") -> {
+			val host = uri.host.orEmpty()
+			if (host == "f-droid.org" || host.endsWith(".f-droid.org")) {
 				uri.lastPathSegment?.nullIfEmpty()
-			} else null
+			} else if (host == "apt.izzysoft.de") {
+				uri.lastPathSegment?.nullIfEmpty()
+			} else {
+				null
+			}
+		}
+		else -> {
+			null
 		}
-		else -> null
 	}
-}
\ No newline at end of file
+}
diff --git a/droidify.png b/droidify.png
index 1767f5ce..e6650fe2 100644
Binary files a/droidify.png and b/droidify.png differ
diff --git a/release/LeOS-Droid_v1.6.1.apk b/release/LeOS-Droid_v1.6.1.apk
index 7cbaec8b..3e5161ac 100644
Binary files a/release/LeOS-Droid_v1.6.1.apk and b/release/LeOS-Droid_v1.6.1.apk differ
diff --git a/release/output-metadata.json b/release/output-metadata.json
deleted file mode 100644
index 705fafc4..00000000
--- a/release/output-metadata.json
+++ /dev/null
@@ -1,20 +0,0 @@
-{
-  "version": 3,
-  "artifactType": {
-    "type": "APK",
-    "kind": "Directory"
-  },
-  "applicationId": "com.leos.droidify",
-  "variantName": "release",
-  "elements": [
-    {
-      "type": "SINGLE",
-      "filters": [],
-      "attributes": [],
-      "versionCode": 56,
-      "versionName": "1.6.1",
-      "outputFile": "app-release.apk"
-    }
-  ],
-  "elementType": "File"
-}
\ No newline at end of file
